### PR TITLE
[Transformers] All PW transformers, inputs, and outputs

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/16386
Merged w/ https://github.com/department-of-veterans-affairs/vets-website/pull/15131

**Node - QA**
- [x] transformer
- [x] input schema
- [x] output schema

**Node - Step by step**
- [x] transformer  
- [x] input schema
- [x] output schema

**Paragraph alert_single**
- [x] transformer
- [x] input schema
- [x] output schema

**Paragraph rich_text_char_limit_1000**
- [x] transformer
- [x] input schema
- [x] output schema

**Paragraph contact_information**
- [x] transformer
- [x] input schema
- [x] output schema

**Paragraph button**
- [x] transformer
- [x] input schema
- [x] output schema

**taxonomy lc_categories**
- [x] transformer
- [x] input schema
- [x] output schema

**paragraph-step_by_step**
- [x] transformer
- [x] input schema
- [x] output schema

**paragraph-step**
- [x] transformer
- [x] input schema __todo__
- [x] output schema

I think paragraph `audience_topics` and node `landing_page` are already done

## Testing done
Looked at data structures outputted by:
- [x] `node script/cms/test-bundle-transformers.js --entity node.79108f2a-e3be-41e8-aa46-04b8cb0007f6.json`
- [x] ` node script/cms/test-bundle-transformers.js --entity node.2edbc8f7-4259-4396-8633-aa102a889f51.json`
- [x] `yarn build:content --use-cms-export` 
- [x] Capture diffs and resolve discrepancies 

Compared those data structures against the data structures in template, on staging.va.gov (in the console), etc.

Still need to run diffs.

## Notes
- Probably need to update `transformers/paragraph-phone_number.js`, which doesn't return a nested `entity` property.
- I changed `transformers/taxonomy_term-lc_categories.js` to return a nested `entity` property, but IDK what side effects that had.

#### Example of Node QA
`http://localhost:8080/resources/what-if-i-dont-want-a-fiduciary-anymore/`

image

#### Example of Step by step

`http://localhost:8080/resources/how-to-change-direct-deposit-information-for-va-disability-or-pension/`

It's missing the images but we can do that later because this PR is already massive.

image

## Acceptance criteria
- [ ] The transformer versions of R&S articles "Question and answer" and "Step by step" align with the GraphQL versions

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/transformers/paragraph-button.js
Comment (ncksllvn) [inline src/site/stages/build/process-cms-exports/transformers/paragraph-button.js:13]: IDK why the content export representing the button has `uri` whereas the GraphQL version has `path`

**Comment by jhonnyoddball** [comment]
@ncksllvn  I don't see an input schema nor transformer file for `paragraph-email_contact`.
Is this a transformer PW owns? If so, we are going to need it before merging

I also see the following items missing:
```
Missing input schema for paragraph-email_contact
No transformer for target_id paragraph-email_contact
Missing input schema for node-faq_multiple_q_a
Missing output schema for node-faq_multiple_q_a
Missing input schema for node-support_resources_detail_page
Missing output schema for node-support_resources_detail_page
Missing input schema for node-basic_landing_page
Missing output schema for node-basic_landing_page
```
Does PW own those transformers? can you please confirm? If so, they are still missing.

Lastly, can you please update `src/site/stages/build/process-cms-exports/transformers/README.md` with the list of the transformers owned by PW? This way we will be able to tell easier who is the owner. Thank you!

**Comment by ncksllvn** [comment]
@jhonnyoddball the intent of this PR was only to implement Node QA and Node StepByStep, and to pick up required paragraphs as we went. This does not intend to implement _all_ PR transformers because that would make this PR gargantuan. 

> I don't see an input schema nor transformer file for paragraph-email_contact.

I don't know but likely.

> Does PW own those transformers? can you please confirm? If so, they are still missing.

Yes, we own all of those, but again, it isn't this intent of this PR to implement all of them.

> Lastly, can you please update src/site/stages/build/process-cms-exports/transformers/README.md with the list of the transformers owned by PW? This way we will be able to tell easier who is the owner. Thank you!

I still don't really know the extent of all the transformers we own just yet, and I don't feel like waiting for another CI to run to update the README.

**Comment by ncksllvn** [comment]
@jhonnyoddball To be clear, are you saying we need to implement every single Public Websites transformer here in order for us to merge this? This PR is already close to a thousand lines. I would prefer us to coming to a stopping point sooner rather than later and merging this.

File: .github/CODEOWNERS
Comment (kelsonic) [inline .github/CODEOWNERS:55]: Open to suggestions on other ways to do this that's not so verbose 🙂

**Review by kelsonic — APPROVED** [review]

File: .github/CODEOWNERS
Comment (mpelzsherman) [inline .github/CODEOWNERS:156]: Can we break up the transformers into subfolders to make code ownership easier to manage?

Comment (kelsonic) [reply .github/CODEOWNERS:156]: I like the idea of that, but at the same time our current team structure will likely change into the future so I wonder if it will make it for future teams to find a transformer if we do that. What do you think?

Comment (mpelzsherman) [reply .github/CODEOWNERS:156]: I still think grouping the transformers in some way would make sense. We have a similar issue with the liquid templates.

**Comment by kelsonic** [comment]
Closing in favor of https://github.com/department-of-veterans-affairs/vets-website/pull/15356

**Pull request closed without merging**
